### DISCUSSION TITLE
How does is_changed() work?

### DISCUSSION THREAD

**Discussion by KirmesBude** [bevyengine/bevy#8237] [Q&A] [answered]
Hi,

I wanted to create a bevy plugin for 2d animations that very closely mirrors the bevy AnimationPlayer and I am running into an issue with is_changed().
I check the value here https://github.com/KirmesBude/bevy_trickfilm/blob/main/src/animation/animation_spritesheet.rs#L44 and you are able to change it from the example here https://github.com/KirmesBude/bevy_trickfilm/blob/main/examples/spritesheet_animation_titan.rs#L63 (mirrors the animated_fox example).

This as close to the example and behaviour of animation player on bevy, but in my case is_changed() always returns false.

Would be glad for any pointers.

**Comment by tim-blackbird** [comment, marked as answer]
I think this is the change detection bug with par_iter. Fixed by #8029 which will be included in Bevy 0.10.1

**Reply by KirmesBude** [reply]
That does appear to fix the issue. Interestingly I tested animated_fox on the v0.10.0 tag and it worked fine there. 
I will just wait for 0.10.1 then.
